packages/openneuro-app/src/scripts/users/user-query.tsx

```diff
@@ -0,0 +1,81 @@
+// UserQuery.tsx
+
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { UserRoutes } from './user-routes';
+import FourOFourPage from '../errors/404page';
+
+// ORCID validation regex pattern
+const orcidPattern = /^\d{4}-\d{4}-\d{4}-\d{4}$/;
```

Comment:
The ORCID can end in X as the last value is a checksum.

We should reuse this implementation from here:
https://github.com/OpenNeuroOrg/openneuro/blob/08e28d29991668fd032aabbeb1f2185d3b24238c/packages/openneuro-app/src/scripts/dataset/mutations/update-permissions.tsx#L10-L18

Reply:
added new validationUtils.ts that is now available to on-app